main/test/ca/uwaterloo/flix/language/phase/TestWeeder.scala

```diff
@@ -136,36 +136,6 @@ class TestWeeder extends FunSuite with TestUtils {
     expectError[WeederError.EmptyLattice](result)
   }
 
```

Comment:
Here it would be good with just a few negative test cases. Especially for arrays.